iceoryx_hoofs/test/stresstests/test_stress_sofi.cpp

```diff
@@ -157,7 +158,7 @@ TEST_F(SoFiStress, SimultaneouslyPushAndPopOnEmptySoFi)
             isPushing = false;
 
             // if we do not get an expected value, perform the test for logging and stop the threads
-            if (pushResult == false || valOut >= 0)
+            if (!pushResult || valOut >= 0)
```

Comment:
Does clang-tidy like this better? If so, interesting as it has more potential for hidden operations when looking at the line without type information context (could be `operator!` for custom types etc.).

Anyway, I prefer how it was changed.

Reply:
Yes clang-tidy complains about the 'redundant boolean literal supplied to boolean operator'.